os/net/net_initialize_netmgr.c

```diff
@@ -26,7 +26,9 @@ struct tr_netmgr {
 	void *dev;
 };
 
+#ifdef CONFIG_LWNL80211
```

Comment:
I'll add dependency to Kconfig. so please remove this

Reply:
Done.